packages/typescript/internal/ts_project.bzl

```diff
@@ -568,15 +569,19 @@ def ts_project_macro(
 
             When a custom transpiler is used, then the `ts_project` macro expands to these targets:
 
-                - `[name]` - the default target is a `js_library` which can be included in the `deps` of downstream rules.
-                    Note that it will successfully build *even if there are typecheck failures* because the `tsc` binary
-                    is not needed to produce the default outputs.
-                    This is considered a feature, as it allows you to have a faster development mode where type-checking
-                    is not on the critical path.
-                - `[name]_typecheck` - this target will fail to build if the type-checking fails, useful for CI.
-                - `[name]_typings` - internal target which runs the binary from the `tsc` attribute
-                -  Any additional target(s) the custom transpiler rule/macro produces.
-                    Some rules produce one target per TypeScript input file.
+            - `[name]` - the default target is a `js_library` which can be included in the `deps` of downstream rules.
+                Note that it will successfully build *even if there are typecheck failures* because the `tsc` binary
+                is not needed to produce the default outputs.
+                This is considered a feature, as it allows you to have a faster development mode where type-checking
+                is not on the critical path.
+            - `[name]_typecheck` - this target will fail to build if the type-checking fails, useful for CI.
```

Comment:
hmm. technically the build failure will be in the `[name]_typings` `ts_project` target and not the `_typecheck` target

Reply:
fixed the wording